examples/graphics/render-to-cubemap.html

```diff
@@ -0,0 +1,230 @@
+        shinyBall.setLocalScale(10, 10, 10);
+
+        // get world and skybox layers
+        var worldLayer = app.scene.layers.getLayerByName("World");
+        var skyboxLayer = app.scene.layers.getLayerByName("Skybox");
+
+        // add camera component to shiny ball - this defines camera properties for cubemap rendering 
+        shinyBall.addComponent('camera', {
+
+            // optimization - no need to clear as all pixels get overwritten
+            clearColorBuffer: false,
+
+            // cubemap camera will render objects on world layer and also skybox
+            layers: [worldLayer.id, skyboxLayer.id],
+
+            // priority - render before world camera
+            priority: -1,
+
+            // disable as this is not a camera that renders cube map but only a container for properties for cube map rendering
+            enabled: false
+        });
+        
+        // add cubemapRenderer script component which takes care of rendering dynamic cubemap
+        shinyBall.addComponent('script');
+        shinyBall.script.create('cubemapRenderer', {
+            attributes: {
+                resolution: 256,
+                mipmaps: true,
+                depth: true
+            }
+        });
+
+        // finish set up of shiny material - make reflection a bit darker
+        shinyMat.diffuse = new pc.Color(0.6, 0.6, 0.6);
+
+        // use cubemap which is generated by cubemapRenderer instead of global skybox cubemap
+        shinyMat.useSkybox = false;
+        shinyMat.cubeMap = shinyBall.script.cubemapRenderer.cubeMap;
+
+        // make it shiny without diffuse component        
+        shinyMat.metalness = 1;
+        shinyMat.useMetalness = true;
+        shinyMat.update();
+
+        // create few random primitives in the world layer
+        var entities = [];
+        var shapes = ["box", "cone", "cylinder", "sphere", "capsule"];
+        for (var i = 0; i < 6; i++) {
+            var shapeName = shapes[Math.floor(Math.random() * shapes.length)];
+            var color = new pc.Color(Math.random(), Math.random(), Math.random());
+            entities.push(createPrimitive(shapeName, pc.Vec3.ZERO, new pc.Vec3(3, 3, 3), color, [worldLayer.id]));
+        }
+
+        // create green plane as a base to cast shadows on
+        createPrimitive("plane", new pc.Vec3(0, -8, 0), new pc.Vec3(20, 20, 20), new pc.Color(0.3, 0.5, 0.3), [worldLayer.id]);
+
+        // Create main camera, which renders entities in world, excluded and skybox layers
+        var camera = new pc.Entity("MainCamera");
+        camera.addComponent("camera", {
+            fov: 60,
+            layers: [worldLayer.id, excludedLayer.id, skyboxLayer.id]
+        });
+        app.root.addChild(camera);
+
+        // Create an Entity with a directional light component
+        var light = new pc.Entity();
+        light.addComponent("light", {
+            type: "directional",
+            color: new pc.Color(1, 1, 1),
```

Comment:
```suggestion
            color: pc.Color.WHITE,
```

Reply:
changed it to YELLOW to give more happy feel ;)